f2/src/forms/WhatHappenedForm.js

```diff
@@ -28,45 +19,61 @@ export const WhatHappenedForm = (props) => {
 
   let formLabel = <Trans id="whatHappenedPage.summary" />
   let formHelpText = <Trans id="whatHappenedPage.hint" />
-  let nextButton = <Trans id="whatHappenedPage.nextPage" />
+  let nextPage = <Trans id="whatHappenedPage.nextPage" />
   let formHelpText2
 
   if (fyiForm) {
     formLabel = <Trans id="whatHappenedPage.fyi.summary" />
     formHelpText = <Trans id="whatHappenedPage.fyi.hint" />
     formHelpText2 = <Trans id="whatHappenedPage.fyi.hint2" />
-    nextButton = <Trans id="fyiForm.nextPage2" />
+    nextPage = <Trans id="fyiForm.nextPage2" />
   }
 
   return (
-    <Form
+    <Formik
       initialValues={whatHappened}
-      onSubmit={localOnSubmit}
-      render={({ handleSubmit }) => (
-        <Stack as="form" onSubmit={handleSubmit} shouldWrapChildren>
-          <FormLabel htmlFor="whatHappened">{formLabel}</FormLabel>
-          <FormHelperText htmlFor="whatHappened">
-            {formHelpText}
-            {formHelpText2 ? (
-              <p>
-                <br />
-                {formHelpText2}
-              </p>
-            ) : null}
-          </FormHelperText>
-          <Field
-            name="whatHappened"
-            component={TextArea}
-            h="300px"
-            marginTop="-1.5rem"
-          />
-          <NextAndCancelButtons
-            next={nextButton}
-            button={<Trans id="whatHappenedPage.nextButton" />}
-          />
-        </Stack>
+      onSubmit={(values) => {
+        props.onSubmit(values)
+      }}
+    >
+      {({ handleSubmit, handleChange, handleBlur }) => (
+        <Form onSubmit={handleSubmit}>
+          <Container>
+            <Row className="form-question">
+              <Row className="form-label">
+                {formLabel}
+              </Row>
+              <Row className="form-helper-text">
+                {formHelpText}
+                {formHelpText2 ? (
+                <p>
+                  <br />
+                  {formHelpText2}
+                </p>
+                ) : null}
+              </Row>
+            </Row>
+            <Row className="form-section">
+              <Field
+                name="whatHappened"
+                component={TextArea}
+                onChange={handleChange}
+                onBlur={handleBlur}
+                rows="12"
```

Comment:
The rows property does not take effect. Rows is an HTML attribute and will not be rendered into a proper CSS rule. To get a large TextArea pass the prop size="large".

Reply:
It will if you pass it to Form.Control. I tested.
I know you add 'large-text-area', but that is a fixed height (300px). It is more flexible to pass props so that you can define any height for the textArea by "rows", and potentially in the future, we could pass in other props for the textArea to make this component more flexible.

Reply:
300px was the height used previously with Chakra-UI components. 

Since the prop is being passed after all please remove the largeTextArea code from TextArea component. In the future we will likely use Emotion to style the components so this class was simply a stop-gap anyway.

Reply:
OK, I removed largeTextArea as you suggested.